src/librustc_typeck/check/upvar.rs

```diff
@@ -141,7 +141,7 @@ impl<'a, 'tcx> FnCtxt<'a, 'tcx> {
             // This gives us an easier access to them without having to call
             // tcx.upvars again..
             if !upvar_list.is_empty() {
-                self.tables.borrow_mut().upvar_list.insert(closure_def_id, upvar_list);
+                self.tables.borrow_mut().closure_captures.insert(closure_def_id, upvar_list);
```

Comment:
I think the variable `upvar_list` should be renamed to `closure_captures`

Reply:
From what I understand this is the type-checking phase, that is where we decide what is the exact type/ places being captured.

Reply:
“captures” or “captured paths” will refer to the paths captured by the closure (e.g., x.foo). This list is produced by the type check.

Reply:
Renamed. Running tests again, they should pass